### PR TITLE
New fields for Rules and Checks with bonus Alembic migrations!

### PR DESCRIPTION
**Pull request by chall8908**
Added a `name` field to both Rules and Checks (per our discussion).

As part of testing, I ended up going ahead with setting up Alembic so that I could actually modify the database in a sane way.

### CODE REVIEW

File: src/Checks.js [hunk 1]
```diff
@@ -90,18 +87,24 @@ class CheckForm extends Component {
...
+                    placeholder={placeholder}
+                    key='control'
+                />);
 
     return (
       <ResourceForm data={this.state} baseResource={this.props.baseResource}>
+        <h2>New Check</h2>
+        <FormGroup controlID="checkName">
+          <ControlLabel>Name</ControlLabel>
+          <FormControl value={this.state.check_name} placeholder="Enter a name for your check" />
```
Comment (bhgames) [inline src/Checks.js]: Question - how does this in turn change the check_name on this.state without an onChange? and why not just use the SingleFieldElement wrapper?

File: src/General.js [hunk 2]
```diff
@@ -68,10 +73,10 @@ export class WithData extends Component {
         })
       );
 
-      return <div>{childrenWithProps}</div>
-    } else {
-      return <div>Loading...</div>
+      return <div className="container">{childrenWithProps}</div>
```
Comment (bhgames) [inline src/General.js:76]: why the className addition? Curious.

**Review by bhgames — COMMENTED** [review]
Had a question or two. Did you make sure tests pass with nosetests --nologcapture?

File: src/General.js (hunk 2 above)
Comment (chall8908) [reply src/General.js:76]: Makes the page look nicer.

File: src/Checks.js (hunk 1 above)
Comment (chall8908) [reply src/Checks.js]: In response to your first question, we always used Redux-Form which handled all of that for us.  Didn't really stop to see if you were using it.

As to your second, I didn't really know what that did and the rest seemed to have some kind of extra logic attached to them to do special shit (like show and hide).  I figured that's what those were for.

**Comment by chall8908** [comment]
I can't run the tests.  I get errors trying to connect to "Impala".

**Comment by bhgames** [comment]
@chall8908 you need to install the MapR 5.2 sandbox, as in the Development instructions. Run it with VirtualBox, impala runs on it.

**Pull request merged**